airbyte-webapp/src/packages/cloud/views/credits/CreditsPage/components/RemainingCredits.tsx

```diff
@@ -83,6 +83,12 @@ const RemainingCredits: React.FC<Props> = ({ selfServiceCheckoutEnabled }) => {
         }, 3000);
       }
     }
+
+    return () => {
+      if (retryIntervalId.current) {
+        clearInterval(retryIntervalId.current);
+      }
+    };
```

Comment:
Fun fact: You don't need to check if it's even set, clearInterval is happy with any input!
`return () => clearInterval(retryIntervalId.current);`

Reply:
Thanks for the tip!